### PR TITLE
Release 0.15.0

### PR DESCRIPTION
**Pull request by mrocklin**
I plan to cut a release soon.  This PR serves as an early warning for those watching the repository.  Is there anything critical that must be addressed before releasing?

### CODE REVIEW

**Comment by jcrist** [comment]
Maybe we should bump pandas on travis to the new 0.20.2 release, just in case?

**Comment by mrocklin** [comment]
Done

**Comment by mrocklin** [comment]
Numpy 1.13.0 was released.  Blocking on fixing things for that.

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,37 @@
...
 
+0.15.0 / 2017-06-09
+-------------------
+
+Array
++++++
+
+-  Add dask.stats module (:pr:`2269`)
+-  Support ``ufunc.outer`` (:pr:`2345`)
+-  Optimize fancing indexing by reducing graph overhead (:pr:`2333`) (:pr:`2394`)
```
Comment (jakirkham) [inline docs/source/changelog.rst]: fancing -> fancy ?

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
+-  Faster array tokenization using alternative hashes (:pr:`2377`)
```
Comment (jakirkham) [inline docs/source/changelog.rst]: Probably worth including some mention of `matmul`.

Comment (jakirkham) [inline docs/source/changelog.rst]: Also we now support all functions in `numpy.fft`. Some like `fftshift` have been requested by others before. So probably worth noting.

**Pull request merged**